test/Microsoft.Dnx.Tooling.FunctionalTests/DnuPublishTests2.cs

```diff
@@ -15,6 +17,31 @@ namespace Microsoft.Dnx.Tooling.FunctionalTests
     [Collection(nameof(ToolingFunctionalTestCollection))]
     public class DnuPublishTests2 : DnxSdkFunctionalTestBase
     {
+        [ConditionalTheory]
+        [OSSkipCondition(OperatingSystems.Linux | OperatingSystems.MacOSX)] // This needs msbuild to run
+        [MemberData(nameof(DnxSdks))]
+        public void PublishWrappedProjectForSpecificFramework(DnxSdk sdk)
+        {
+            var solutionName = Path.Combine("DnuWrapTestSolutions", "WrapAndPublish");
+            var solution = TestUtils.GetSolution<DnuPublishTests2>(sdk, solutionName);
+
+            // Restore the wrapper project
+            sdk.Dnu.Restore(solution.GetWrapperProjectPath("ClassLibrary")).EnsureSuccess();
+
+            // Publish the console app
+            sdk.Dnu.Publish(solution.GetProject("DnxConsoleApp").ProjectFilePath, solution.ArtifactsPath, "--framework dnx451").EnsureSuccess();
+
+            // Get an SDK we can use the RUN the wrapped project (it has to be CLR because the project only supports dnx451)
+            var clrSdk = GetRuntime("clr", "win", "x86");
+
+            // Run it
+            var outputPath = Path.Combine(solution.ArtifactsPath, "approot", "src", "DnxConsoleApp");
+            var result = clrSdk.Dnx.Execute(
```

Comment:
No because the batch file relies on the `dnx` on the path. I could try fiddling with putting one on the path but instead I wrote this to do exactly the same invocation that the batch file does, but with a specific DNX.